src/pybind/mgr/progress/module.py

```diff
@@ -24,19 +23,20 @@ class Event(object):
     objects (osds, pools) this relates to.
     """
 
-    def __init__(self, message, refs, started_at=None):
+    def __init__(self, message, refs, started_at=None, add_to_ceph_s=False):
```

Comment:
let's default add_to_ceph_s to true for now, we can start excluding events if there are too many later